src/util/utils.js

```diff
@@ -18,6 +18,14 @@ import { getWalletFiat } from '../util/CurrencyWalletHelpers.js'
 export const DECIMAL_PRECISION = 18
 export const DEFAULT_TRUNCATE_PRECISION = 6
 
+export function isoFiatSplatter(cc: string): { isoFiatCurrencyCode: string, fiatCurrencyCode: string } {
```

Comment:
We already have a function that adds an iso if needed: https://github.com/EdgeApp/edge-react-gui/blob/64c512fe17bc196aa8b1e9aed9215a719967edd3/src/util/utils.js#L273
And removing it is just `code.replace('iso:', '')` so not sure that we even need a function for that.

Reply:
We are very inconsistent in the use of fiatCurrencyCode vs isoFiatCurrencyCode. Especially when used in the core versus the GUI Wallet. This makes it really easy to accidentally use one versus another.

The new routine is the most flexible in that it can take either format and output both formats in one line.

Reply:
This doesn't help and just adds extra code and unnecessary complexity.

You shifted the problem from needing to know which function to call to needing knowing which value you need to destruct from the returned value.

You still have the exact same problem only with an extra step and in a more implicit way which actually makes it actually harder to debug and find errors.

If you really want to solve the issue, then we need to only use the iso code in the GUI and then we won't even an issue.